packages/editor/src/client/locales/en.json

```diff
@@ -618,6 +620,7 @@
         "close": "Close",
         "confirm": "Confirm",
         "createdAt": "{{createdAt, ccc PP p}}",
+        "dateExplanationPopOver": "This message will explain precisely what the 'available online from' date does.",
```

Comment:
Can we change this? :smile:

Reply:
I'm waiting for Nina's update, she'll write a non-technical sentence about this